services/utils/__init__.py

```diff
@@ -171,8 +174,10 @@ def __init__(self,
         self.database_name = os.environ.get(
             prefix + "NAME", database_name).translate(table)
 
-        self.pool_min = int(os.environ.get(prefix + "POOL_MIN", self.pool_min))
-        self.pool_max = int(os.environ.get(prefix + "POOL_MAX", self.pool_max))
+        self.pool_min = int(os.environ.get(prefix + "POOL_MIN", pool_min))
+        self.pool_max = int(os.environ.get(prefix + "POOL_MAX", pool_max))
```

Comment:
init doesn't seem to take pool_min/pool_max as inputs currently so these are unassigned